resources/user.py

```diff
@@ -224,3 +227,23 @@ def get(self):
             } for user in users]
 
         return {"users": ret}, 200
+
+class UserInvite(Resource):
+    @jwt_required
+    def post(self):
+        data = request.json
+        temp_password = ''.join(random.choice(string.ascii_lowercase) for i in range(8))
+        user = UserModel(
+            email=data["email"],
+            firstName=data["firstName"],
+            lastName=data["lastName"],
+            phone=data["phone"],
+            password=temp_password,
+            role=RoleEnum.STAFF,
```

Comment:
Should this be a hard-coded role?

Reply:
Changed this to accept whatever value is in the JSON. My confusion was that I thought non-admin users might be inviting other users, so I wanted to set it at a lower level.

Reply:
That makes sense, however the requirement is that only admins can invite users.